src/pudl/metadata/resources/eia861.py

```diff
@@ -492,6 +623,12 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_reliability": {
+        "description": (
+            """The system average interruption duration index (SAIDI), system average
```

Comment:
A first sentence just describing this table conceptually would be helpful, after which we can list the specific metrics as you have here. Maybe:

```
"""Standardized metrics of electricity system reliability and outage impacts.

Includes the system average interruption duration index (SAIDI), system average
interruption frequency index (SAIFI), and customer average interruption duration index
(CAIDI) aka SAIDI/SAIFI with and without major event days and loss of service. Includes
the standard (IEEE/other) and other relevant information."""
```